### DISCUSSION TITLE
Conflicting dependencies while installing

### DISCUSSION THREAD

**Discussion by leonora-sewidan** [Ulm-IQO/qudi-core#84] [Q&A] [answered]
I am using Python 3.9.10 and trying to install the qudi-iqo-modules. While running **python -m pip install -e .** I am getting this error about conflicting dependencies:

_ERROR: Cannot install qudi-core==1.5.0 and qudi-iqo-modules==0.5.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    qudi-iqo-modules 0.5.0 depends on PySide2==5.15.2
    qudi-core 1.5.0 depends on PySide2==5.15.2.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies_

How can I resolve this issue? I have already updated to the newest version as specified in the guide with 
                                      **python -m pip install git+https://github.com/Ulm-IQO/qudi-core.git@main**

**Comment by timoML** [comment]
We are currently in the process of releasing v0.5.0 and in the meantime qudi-iqo-modules is hanging apparently in an uninstallable state. I am very sorry, this should not happen.

For now, I suggest to install the latest stable release via
    
    cd C:/Software/qudi-iqo-modules
    git checkout tags/v0.4.0
    python -m pip install -e .

We will try to finish the release by the end of the day. Then, checking out from the latest main will work again.

**Reply by leonora-sewidan** [reply]
Ah okay, so if I try the same that I did again tomorrow it should work?

**Reply by timoML** [reply, marked as answer]
I just fixed the dependency error. It should work now, if you deleted the qudi-iqo-modules folder and repeat `git clone https://github.com/Ulm-IQO/qudi-iqo-modules.git`.

**Reply by leonora-sewidan** [reply]
Thank you, it worked now.
